components/search/Search.vue

```diff
@@ -210,6 +210,10 @@ export default class Search extends mixins(
     return this.preferencesStore.getReplaceBuyNowWithYolo
   }
 
+  get isSearchPage() {
+    return this.routePathList.indexOf(this.$route.path) !== -1
```

Comment:
```suggestion
    return this.routePathList.includes(this.$route.path)
```
easier to read no?

also, maybe calling the function ```isExplorePage``` is a more fitting name?

Reply:
Agree 
a70256b

Reply:
> also, maybe calling the function isExplorePage is a more fitting name?

Yes, updated it